@xen-orchestra/web/src/components/pool/PoolNetworksTable.vue

```diff
@@ -10,8 +10,8 @@
     </UiTitle>
     <div class="container">
       <div class="table-actions">
-        <UiQuerySearchBar @search="(value: string) => (searchQuery = value)" />
-        <UiTableActions :title="$t('table-actions')">
+        <UiQuerySearchBar @search="value => (searchQuery = value)" />
+        <UiTableActions :title="t('table-actions')">
```

Comment:
To keep consistency with the rest of usages:
```suggestion
        <UiTableActions :title="$t('table-actions')">
```